content/kb/troubleshooting-longhorn-manager-crashes-due-to-backing-image-eviction.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Troubleshooting: Longhorn Manager Crashes Due To Backing Image Eviction"
+authors:
+- "Jack Lin"
+draft: false
+date: 2025-03-07
+versions:
+- "v1.7.0-v1.7.3 and v1.8.0-v1.8.1"
```

Comment:
Should be fixed in v1.8.1?
 
```suggestion
- "v1.7.0-v1.7.3 and v1.8.0"
```

Reply:
Oh yeah, I forgot we merged it before releasing 1.8.1...

Reply:
fixed.